ultralytics/utils/replace.json

```diff
@@ -0,0 +1,109 @@
+{
```

Comment:
Is there an extra step that needs to take place to include .json files in the pip package?

When i try installing this branch to test it, i noticed all the .py files are updated but `replace.json` and `post_replace.json` are missing.

I'm still new to the python ecosystem, but If i had to guess the `MANIFEST.in` needs to be updated.

Reply:
@Zoxive I am not sure if we are supposed to answer this question. If so, can you explain how you are installing this branch for testing and what tests you are running?

Reply:
I installed by `pip install git+https://github.com/DeGirum/ultralytics_yolov8.git@franklin_current`

I was able to successfully train a custom model with the new `relu6-yolov8.yaml`
When i attempted to export it with `format=tflite` it would complain that the `replace.json` was missing.

When then i double checked that i did actually have all the changes, and i manually checked out your banch into another folder and did a folder comparision with where pip installed the branch. I noticed that only the *`.json` files were missing.

Reply:
@Zoxive Thanks for the details. We will check to see if adding the files to manifest helps or not. May I ask if you are the assigned reviewer for the pull request?

Reply:
Nope. I'm just a bystander. I was just recently trying to export a tflite model and had problems, so i thought i would try out your branch especially to see if i could see the quantization improvements.